app/scripts/services/rowMethods.js

```diff
@@ -0,0 +1,51 @@
+'use strict';
+
+angular.module('openshiftConsole')
+  .factory('rowMethods', function() {
```

Comment:
The only odd thing is `setInitialExpandedState` takes a "row" and sets `row.expanded` and `row.state.expandAll`.  Hmm.